src/utils/api.js

```diff
@@ -40,4 +44,10 @@ export const postNewComment = (articleId, comment, username) => {
     console.log("article=",articleId, "comment=", comment,"username=", username)
```

Comment:
no console.logs! Submitting a pull request is like saying "I think my code is production ready"